### PR TITLE
474 hide avatar checkbox

### PR DESCRIPTION
**Pull request by radoslawroszkowiak**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/474
**Reference person:** @yourgithubname
**Description:**

Description of your changes.

**New imports / dependencies:**
- `package`

**Unit Tests:**
- `apps/volontulo/tests/views/test_auth.py`

**What tests do I need to run to validate this change:**

Description of testing workflow and potential affected places.

### CODE REVIEW

File: apps/volontulo/tests/forms/test_edit_profile.py [hunk 1]
```diff
@@ -83,3 +83,14 @@ def test__is_validate__distinct_new_passwords__raise_exception(self):
         )
         form = EditProfileForm(data=form_data)
         self.assertRaises(ValidationError, form.is_valid)
+
+    def test__no_is_avatar_field_on_edit_profile_form(self):
+        u"""Test if Is Avatar field is not visible."""
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/forms/test_edit_profile.py:88]: Its unicode already, please remove u"

**Comment by tomaszwozniak** [comment]
minor comment, rest looks :+1:

File: apps/volontulo/tests/forms/test_edit_profile.py (hunk 1 above)
Comment (radoslawroszkowiak) [inline apps/volontulo/tests/forms/test_edit_profile.py:88]: It seems this is the "global" convention followed here. I don't want to break it.

Comment (tomaszwozniak) [inline apps/volontulo/tests/forms/test_edit_profile.py:88]: its wrong convention :trollface:

Comment (radoslawroszkowiak) [inline apps/volontulo/tests/forms/test_edit_profile.py:88]: If we support python2 (Tomek said some time ago we did) we can survive leaving it as it is ;)
BTW róbcie ficzery a nie tu o głupotach ;)

Comment (tomaszwozniak) [inline apps/volontulo/tests/forms/test_edit_profile.py:88]: http://python-future.org/unicode_literals.html FTW!

Comment (radoslawroszkowiak) [inline apps/volontulo/tests/forms/test_edit_profile.py:88]: Unicode handling refactoring is not a part of this ticket. As well as adding unicode_literals.

**Pull request merged**